tests/integration/test_schema_backup_avro_export.py

```diff
@@ -0,0 +1,98 @@
+"""
+karapace - test schema backup
+
+Copyright (c) 2019 Aiven Ltd
+See LICENSE for details
+"""
+from karapace.config import set_config_defaults
+from karapace.schema_backup import SchemaBackup
+from karapace.utils import Client
+from pathlib import Path
+from tests.utils import KafkaServers
+from typing import Any, Dict
+
+import os
+import ujson
+
+baseurl = "http://localhost:8081"
+
+
+JSON_SUBJECT = "json-schemas"
+JSON_SUBJECT_HASH = "a2a0483c6ce0d38798ef218420e3f132608dbebf"
+JSON_SCHEMA = {
+    "type": "object",
+    "title": "JSON-schema",
+    "description": "example",
+    "properties": {"test": {"type": "integer", "title": "my test number", "default": 5}},
+}
+
+AVRO_SUBJECT = "avro-schemas"
+AVRO_SUBJECT_HASH = "a801beafef1fb8c03907b44ec7baca341a58420d"
+AVRO_SCHEMA = {
+    "type": "record",
+    "namespace": "io.aiven",
+    "name": "myrecord",
+    "fields": [
+        {
+            "type": "string",
+            "name": "f1",
+        },
+    ],
+}
+EXPECTED_AVRO_SCHEMA = {
+    "type": "record",
+    "namespace": "aa258230180d9c643f761089d7e33b8b52288ed3.ae02f26b082c5f3bc7027f72335dd1186a2cd382",
+    "name": "afe8733e983101f1f4ff50d24152890d0da71418",
+    "fields": [
+        {
+            "type": "string",
+            "name": "a09bb890b096f7306f688cc6d1dad34e7e52a223",
+        },
+    ],
+}
+
+
+async def insert_data(c: Client, schemaType: str, subject: str, data: Dict[str, Any]) -> None:
+    schema_string = ujson.dumps(data)
+    res = await c.post(
+        "subjects/{}/versions".format(subject),
+        json={"schema": f"{schema_string}", "schemaType": schemaType},
+    )
+    assert res.status == 200
+    assert "id" in res.json()
+
+
+async def test_export_anonymized_avro_schemas(
+    registry_async_client: Client, kafka_servers: KafkaServers, tmp_path: Path
+) -> None:
+    await insert_data(registry_async_client, "JSON", JSON_SUBJECT, JSON_SCHEMA)
```

Comment:
The JSON schema is not anonymized, it is skipped. Insert this data for asserting in the end of the test that it is not processed.